misc_func.py

```diff
@@ -34,15 +36,6 @@ def point_matrix(cols, rows, ind):
             return [Scalar(0) for _ in range(cols)]
 
 
-def key_matrix(cols, rows):
-    rv = [Scalar(0) * dumber25519.G] * cols
-    rh = [Scalar(0) * dumber25519.G] * rows
-
-    for i in range(0, cols):
-        rv[i] = PointVector(rh)
-    return rv
-
```

Comment:
The results of this function were never actually used. It was used just for creating an array of arrays but then every single value was always rewritten.
I think there are more such examples, I can take a look more, and clean up the code a little bit (and gain even more performance).